solutions/devicesimulation/single-vm/setup.sh

```diff
@@ -50,6 +50,7 @@ while [ "$#" -gt 0 ]; do
         --aad-appid)               PCS_WEBUI_AUTH_AAD_APPID="$2" ;;
         --aad-tenant)              PCS_WEBUI_AUTH_AAD_TENANT="$2" ;;
         --aad-instance)            PCS_WEBUI_AUTH_AAD_INSTANCE="$2" ;;
+        --solution-id)           PCS_SOLUTION_ID:"$2" ;;
```

Comment:
Same comment about the ":" as above. Also style nit on indentation matching the items above.